packages/web-components/src/components/va-date/test/va-date.e2e.ts

```diff
@@ -229,7 +236,7 @@ describe('va-date', () => {
     // Trigger Blur
     await handleYear.press('Tab');
     await page.waitForChanges();
-    expect(date.getAttribute('value')).toBe('--');
+    expect(date.getAttribute('value')).toBeNull();
```

Comment:
We aren't getting the `--` anymore because we're stripping the trailing hyphens.